.github/workflows/scripts/pre-submit.actions/references.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Verify that Action calls that are pinned (not locally called)
+# use the same referemnce as the relealse tag.
+# See reasoning in ./github/actions/README.md
+
+set -euo pipefail
+
+results=$(
```

Comment:
nit: maybe we should check that it's either a semver tag if the PR has 'label#release' or `main` if not. You can maybe copy over the semver utilities from example-package.

Reply:
done. PTAL